### DISCUSSION TITLE
Windows spellcheck install help.

### DISCUSSION THREAD

**Discussion by deepestwaterdiver** [zim-desktop-wiki/zim-desktop-wiki#2638] [Q&A] [answered]
Hi, sorry for asking a question that has seemingly been asked a thousand times before, but what is the recommended process for getting the spell checkers plugin working for windows? I've attempted to follow this guide (https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/2047) but pygi-aio-3.24.1-rev1-setup (I'm also making the presumption here that I've downloaded the correct file, the one with a string of mixed characters at the end) doesn't seem to recognise the python install, which is 3.11.7.

**Comment by tridian-tn** [comment, marked as answer]
Hi. I updated the [wiki guide](https://github.com/jaap-karssenberg/zim-wiki/wiki/Install-spell-checking-on-windows) based on what happened when I followed that guide. That's the same pygi-aio-3.24.1-rev1-setup installer that I used. The only difference I can see is that I had Python 3.11.8 when I got it working. I didn't try 3.11.7, although I'm dubious it'll make a difference. Might be worth a try with 3.11.8 though.

**Reply by tridian-tn** [reply]
I get this when I try to install again:
![image](https://github.com/user-attachments/assets/d1db4aa5-2c44-40cd-a5f8-d2c705ab288f)

I then click Add Path and navigate to c:\Python11 (although c:\Python11 is already in my PATH environment variable, so you probably don't need to find the path manually)

Then I get this:
![image](https://github.com/user-attachments/assets/f0d1c2eb-3c82-493f-9b4c-fa6bcaf75472)

I tick it, and then I'm onto package selection:

![image](https://github.com/user-attachments/assets/cd878659-3fc8-4da6-a659-a189b2301704)

How does this differ with what you see?

**Reply by deepestwaterdiver** [reply]
Pygi, in this case, doesn't seem to recognise my install of python.

**Reply by deepestwaterdiver** [reply]
Is it possibly due to it not being a portable install?

**Reply by deepestwaterdiver** [reply]
Ah, figured out the problem. Python had been installed within a file that PyGI could not get access to.

**Reply by deepestwaterdiver** [reply]
Okay, fixed it. Python was installed within a file that PyGI could not access, so it was not recognised.

**Reply by tridian-tn** [reply]
Nice. Glad it's working
